docs_app/content/navigation.json

```diff
@@ -26,17 +26,21 @@
 			"title": "Overview",
 			"tooltip": "RxJS Overview",
 			"children": [
-                {
+								{
 					"url": "guide/observable",
 					"title": "Observables"
-                },
-                {
+				},
+				{
 					"url": "guide/subscription",
 					"title": "Subscription"
 				},
-                {
+				{
 					"url": "guide/subject",
 					"title": "Subjects"
+				},
+				{
```

Comment:
could you maybe change the order. I think I would put it below observables.

Reply:
I'm happy to change it, but I was trying to retain the order of the old docs http://reactivex.io/rxjs/manual/overview.html